SECURITY.md

```diff
@@ -0,0 +1,16 @@
+# Security policy
+
+## Supported versions
+
+| Version       | Supported          |
+| ------------- | ------------------ |
+| 6.1           | :white_check_mark: |
+| < 6.1         | :x:                |
+
+## Reporting a vulnerability
+
+Please report security vulnerabilities by email to [info@ascii-art.site](mailto:info@ascii-art.site "info@ascii-art.site").
+
+If the security vulnerability is accepted, a patch will be crafted privately
+in order to prepare a dedicated bugfix release as soon as possible (depending
+on the complexity of the fix).
```

Comment:
Why privately? I know we are using the scikit-learn template. But it does not make sense to me.

Reply:
Good point.
fixed in https://github.com/sepandhaghighi/art/pull/269/commits/55a01836f52a30c28d658e2d3ba53563c4b68281